.github/workflows/validate_samples.yml

```diff
@@ -0,0 +1,72 @@
+name: Validate-Samples
+
+on:
+  push:
+    paths:
+      - 'samples/*/TIDES/*'
+      - 'spec/*'
+  pull_request:
+    paths:
+      - 'samples/*/TIDES/*'
+      - 'spec/*'
+  workflow_dispatch:
+  create:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install Frictionless
+        run: pip install frictionless[excel,json]>=5.0.0
+      - name: Find Sample Folders
+        id: sample_folders
+        run: |
+          pwd
+          samples=$(find ./samples -type d -name 'TIDES')
+          echo "::set-output name=samples::$samples"
+      - name: Validate Sample Data
+        id: validation
+        run: |
+          samples=(${{ steps.sample_folders.outputs.samples }})
+          results=""
+          for sample in "${samples[@]}"; do
+            result=$(frictionless validate "$sample/datapackage.json" --json)
+            results+="$result\n"
```

Comment:
This *should* upload the result as an artifact...but it is hard to test w/out example data.

Reply:
>...validate only the changed sample data for a sample data push.

I think we want it to validate all the files each time since if the failure comes from a spec change that is also important to know.

Reply:
My suggestion was to validate all sample data when the spec is changes, but validate only the changed sample data when a change to sample data is what triggers the workflow.

Reply:
right now there is zero sample data so i think we can add this when it is relevant